src/include/sof/audio/eq_iir/iir.h

```diff
@@ -10,6 +10,41 @@
 #ifndef IIR_H
 #define IIR_H
 
+/* Get platforms configuration */
+#include <config.h>
+
+/* If next defines are set to 1 the EQ is configured automatically. Setting
+ * to zero temporarily is useful is for testing needs.
+ * Setting EQ_FIR_AUTOARCH to 0 allows to manually set the code variant.
+ */
+#define IIR_AUTOARCH    1
```

Comment:
What about using kconfig to choose between GENERIC and AUTO?

Reply:
That's a good idea, it would be ease testing code variants. Maybe for some tests sub-menu.

Reply:
Yeap, it can be added in debug or audio components menu, for example if you enable IIR then additional options will pop up